room/compiler-processing/src/main/java/androidx/room/compiler/processing/XMessager.kt

```diff
@@ -37,7 +37,7 @@ abstract class XMessager {
         onPrintMessage(kind, msg, element)
     }
 
-    abstract fun onPrintMessage(kind: Diagnostic.Kind, msg: String, element: XElement? = null)
+    protected abstract fun onPrintMessage(kind: Diagnostic.Kind, msg: String, element: XElement? = null)
```

Comment:
I realized this would pop up in auto complete when trying to print a message, which is confusing and seems error prone. I believe it's use case is only internal

Reply:
probably my bad, thanks.